src/coord/src/coord/prometheus.rs

```diff
@@ -0,0 +1,151 @@
+// Copyright Materialize, Inc. All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
```

Comment:
could you add a module-level comment explaining that this mod injects metrics from the prometheus registry into our metrics tables?